Manual installation on Windows: [ERROR] AbsolutePaths - Could not identity Etherpad base path in this win32 installation in "C:\etherpad-lite\src"

Etherpad manually installed on Windows doesn't work. Node fails to run with this error:
> [ERROR] AbsolutePaths - Could not identity Etherpad base path in this win32 installation in "C:\etherpad-lite\src"

### Steps to repro

1. Log on to Windows 10 1803 x64.

1. Install Node.js.
  https://nodejs.org/dist/v10.15.1/node-v10.15.1-x64.msi
  Accept the default options during the installation.

1. Install Git for Windows.
  https://github.com/git-for-windows/git/releases/download/v2.20.1.windows.1/Git-2.20.1-64-bit.exe
  Accept the default options during the installation.

1. Restart Windows.
  Just to make sure the prerequisite installations take effect.

1. Run `C:\Windows\System32\cmd.exe` as administrator (i.e. UAC elevated).

1. Install Etherpad and run it.
    1. Go to `C:\`
    1. Clone the repository master branch
    1. Go to `C:\etherpad-lite`
    1. Run the installation script (`bin\installOnWindows.bat`)
    1. Run `start.bat`

```console
Microsoft Windows [Version 10.0.17134.1]
(c) 2018 Microsoft Corporation. All rights reserved.

C:\Windows\system32>cd \

C:\>git clone --branch master https://github.com/ether/etherpad-lite.git
Cloning into 'etherpad-lite'...
remote: Enumerating objects: 5, done.
remote: Counting objects: 100% (5/5), done.
remote: Compressing objects: 100% (5/5), done.
remote: Total 31721 (delta 0), reused 1 (delta 0), pack-reused 31716 eceiving objects: 100% (31721/31721), 19.05 MiB | 3Receiving objects: 100% (31721/31721), 20.15 MiB | 3.92 MiB/s, done.

Resolving deltas: 100% (22590/22590), done.

C:\>cd etherpad-lite

C:\etherpad-lite>bin\installOnWindows.bat
_
Ensure that all dependencies are up to date...  If this is the first time you have run Etherpad please be patient.
symbolic link created for ep_etherpad-lite <<===>> ..\src
npm WARN deprecated socks@1.1.10: If using 2.x branch, please upgrade to at least 2.1.6 to avoid a serious bug with socket data flow and an import issue introduced in 2.1.0

> wd@1.11.1 install C:\etherpad-lite\node_modules\ep_etherpad-lite\node_modules\wd
> node scripts/build-browser-scripts

added 1050 packages from 1461 contributors and audited 8186 packages in 121.953s
found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details
_
Clearing cache...
Could Not Find C:\etherpad-lite\var\minified*
_
Setting up settings.json...
Can't find settings.json.
Copying settings.json.template...
        1 file(s) copied.
_
Installed Etherpad!  To run Etherpad type start.bat

C:\etherpad-lite>start.bat

C:\etherpad-lite>node node_modules\ep_etherpad-lite\node\server.js
[2019-02-08 23:32:24.837] [DEBUG] console - Running on Node v10.15.1 (minimum required Node version: 6.9.0)
[2019-02-08 23:32:25.009] [DEBUG] AbsolutePaths - C:\etherpad-lite\src does not end with "node_modules\ep_etherpad-lite"
[2019-02-08 23:32:25.009] [ERROR] AbsolutePaths - Could not identity Etherpad base path in this win32 installation in "C:\etherpad-lite\src"

C:\etherpad-lite>
```

--- a/src/node/utils/AbsolutePaths.js
+++ b/src/node/utils/AbsolutePaths.js
@@ -61,8 +61,8 @@ var popIfEndsWith = function(stringArray, lastDesiredElements) {
  * Heuristically computes the directory in which Etherpad is installed.
  *
  * All the relative paths have to be interpreted against this absolute base
- * path. Since the Unix and Windows install have a different layout on disk,
- * they are treated as two special cases.
+ * path. Since the Windows package install has a different layout on disk, it is
+ * dealt with as a special case.
  *
  * The path is computed only on first invocation. Subsequent invocations return
  * a cached value.
@@ -79,26 +79,27 @@ exports.findEtherpadRoot = function() {
 
   const findRoot = require('find-root');
   const foundRoot = findRoot(__dirname);
+  const splitFoundRoot = foundRoot.split(path.sep);
 
-  var directoriesToStrip;
-  if (process.platform === 'win32') {
+  /*
+   * On Unix platforms and on Windows manual installs, foundRoot's value will
+   * be:
+   *
+   *   <BASE_DIR>\src
+   */
+  var maybeEtherpadRoot = popIfEndsWith(splitFoundRoot, ['src']);
+
+  if ((maybeEtherpadRoot === false) && (process.platform === 'win32')) {
     /*
-     * Given the structure of our Windows package, foundRoot's value
-     * will be the following on win32:
+     * If we did not find the path we are expecting, and we are running under
+     * Windows, we may still be running from a prebuilt package, whose directory
+     * structure is different:
      *
      *   <BASE_DIR>\node_modules\ep_etherpad-lite
      */
-    directoriesToStrip = ['node_modules', 'ep_etherpad-lite'];
-  } else {
-    /*
-     * On Unix platforms, foundRoot's value will be:
-     *
-     *   <BASE_DIR>\src
-     */
-    directoriesToStrip = ['src'];
+    maybeEtherpadRoot = popIfEndsWith(splitFoundRoot, ['node_modules', 'ep_etherpad-lite']);
   }
 
-  const maybeEtherpadRoot = popIfEndsWith(foundRoot.split(path.sep), directoriesToStrip);
   if (maybeEtherpadRoot === false) {
     absPathLogger.error(`Could not identity Etherpad base path in this ${process.platform} installation in "${foundRoot}"`);
     process.exit(1);

